Aplib.Core/Desire/GoalStructures/SequentialGoalStructure.cs

```diff
@@ -49,11 +54,7 @@ public override void UpdateStatus(TBeliefSet beliefSet)
             // This loop is here to prevent tail recursion.
             while (true)
             {
-                if (Status == CompletionStatus.Success)
-                {
-                    return;
-                }
-
+                if (Status == CompletionStatus.Success) return;
```

Comment:
Nit: keep the newline?
```suggestion
                if (Status == CompletionStatus.Success) return;
                
```